spec/models/reservation_spec.rb

```diff
@@ -308,20 +311,30 @@
   end
 
   describe "#canceled?" do
+    # TODO: move to order_detail_spec
```

Comment:
If we can't come up with something, it's fine, but it might be nice if we can think of a better name for our `is_canceled?` method. Something that encompass the fact it might mean canceled with fee (which is technically in complete state).

`manually_canceled`, `actual_canceled?`,`actual_cannibal_shia_lebeouf?`. I don't know. I got nothing.